src/kernel/kernel_main.c

```diff
@@ -29,10 +29,12 @@ void kernel_main(const char *cmdline) {
     // device initialization here
 
     /** PROCESS SCHEDULER INITIALIZATION **/
-    // SVCall has priority 14
-    SCB->SHP[2] |= (14 << 24);
-    // PendSV has priority 15
-    SCB->SHP[3] |= (15 << 16);
+    // SysTick has priority 15
+    SCB->SHP[3] |= (15 << 24);
+    // PendSV has priority 14
+    SCB->SHP[3] |= (14 << 16);
```

Comment:
should this be `SCB->SHP[3]` twice?

Reply:
The intent is to make it more readable, though you're right, this could easily be one lined. Thoughts?

Reply:
separate is good :)